game/game.js

```diff
@@ -410,4 +410,35 @@ function canPlay(baseGame, username){
         let medium = baseGame.mediums.find(medium => medium.username == username);
         return medium.hasPlayed ? false : (baseGame.ghost.mediumsHasCards.find(username => username == medium.username) != undefined);
     }
+}
+
+/**
+ * vérifie que tout les joueurs ont trouvés leur scénario
+ *  @param {object} baseGame Instance de jeu
+ * @returns {boolean} goForTheFinal indique si le scenario final doit être lancé ou non
+ */
+function allScenariosFinded (baseGame){
```

Comment:
Functions are actions and thus should start with a verb

Reply:
The past participle of find is found, not finded

Reply:
Why `baseGame` and not just `game`? Does `base` mean anything specific in this context? If not, remove it.